### PR TITLE
Update samples to beta1

### LINKED ISSUE #260: Update all samples for latest beta

### PR DESCRIPTION
**Pull request by adrianhall**
Fixes #260

### CODE REVIEW

File: samples/todo/azureMobile.js [hunk 1]
```diff
@@ -20,21 +18,22 @@ module.exports = {
     logging: {
         // Override for the Winston transports
         // for more info, see https://github.com/winstonjs/winston/blob/master/docs/transports.md
-        // transports: {
-        // Place any Winston compatible transport here
-        //},
+        transports: [
+            new winston.transports.Console({ colorize: true, timestamp: true })
+        ],
```
Comment (mamaso) [inline samples/todo/azureMobile.js]: This is not currently how this works, however this is super clean.  Looking at winston docs they just added a logger.configure method which solves a bunch of the configuration problems we had before so.. yay.  Keep this, I'll change the code to match.

Comment (mamaso) [inline samples/todo/azureMobile.js]: #262

**Comment by mamaso** [comment]
looks good to me

**Comment by danderson00** [comment]
Is this ready to merge based on the logging configuration changes?

**Comment by adrianhall** [comment]
@mamaso is going to merge this one then update the azureMobile.js in samples/todo to match the new stuff.

**Pull request merged**